lib/Swagger/src/TemplateClassification/Storage/Api/TCSApi.php

```diff
@@ -92,6 +92,87 @@ public function setApiClient(ApiClient $apiClient)
   
     
     /**
+     * getTemplateTypesOnWiki
+     *
+     * Provides template types on wiki
+     *
+     * @param int $wiki_id Wikia ID (required)
+     * @return \Swagger\Client\TemplateClassification\Storage\Models\TemplateTypeHolder
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getTemplateTypesOnWiki($wiki_id)
```

Comment:
Same here - do we get a list of types or a list of classified templates with their types? If the latter, I'd consider renaming the method.